src/Engine/ProtoCore/Lang/CallSite.cs

```diff
@@ -1507,15 +1507,16 @@ private FunctionGroup FirstFunctionGroupInInheritanceChain(RuntimeCore runtimeCo
             StackFrame stackFrame,
             RuntimeCore runtimeCore,
             SingleRunTraceData previousTraceData,
-            SingleRunTraceData newTraceData)
+            SingleRunTraceData newTraceData,
+            FunctionEndPoint finalFuntionEndPoint = null)
```

Comment:
please update the params summary above to match.

Reply:
Is the above summary for this function no longer valid?

```
         /// Excecute an arbitrary depth replication using the full slow path algorithm
```

Reply:
I believe it is still valid